kotlin/edit-with-branch-versioning/src/main/java/com/esri/arcgisruntime/sample/editwithbranchversioning/MainActivity.kt

```diff
@@ -227,15 +227,19 @@ class MainActivity : AppCompatActivity() {
     val serviceVersionInfoFuture =
       serviceGeodatabase.createVersionAsync(serviceVersionParameters)
     serviceVersionInfoFuture.addDoneListener {
-      // get the new version's name and switch to it
-      val serviceVersionInfo = serviceVersionInfoFuture.get()
-      createdVersionName = serviceVersionInfo.name
-      switchVersion(null)
+        try{
+            // get the new version's name and switch to it
+            val serviceVersionInfo = serviceVersionInfoFuture.get()
+            createdVersionName = serviceVersionInfo.name
+            switchVersion(null)
+            // hide the create version button and allow the user to switch versions now
+            createVersionButton.visibility = View.GONE
+            switchVersionButton.visibility = View.VISIBLE
+            }catch (e: java.lang.Exception){
+            Log.e(TAG, e.message.toString())
+            Toast.makeText(this, e.message.toString(),Toast.LENGTH_SHORT).show()
```

Comment:
Could be good to prepend with some plain English explanation of what might have gone wrong here. "Error getting service info: " + e.message, or something like that